tests/integration/security/sds_ingress/quic/ingress_test.go

```diff
@@ -96,6 +101,11 @@ func TestMtlsGatewaysWithQUIC(t *testing.T) {
 		NewTest(t).
 		RequiresSingleCluster().
 		Run(func(t framework.TestContext) {
+			err := ingressutil.WaitForIngressQUICService(t, istio.DefaultSystemNamespace)
```

Comment:
Instead of repeating this check on every test, is it possible or better to move it to `TestMain()`?

Reply:
Done

Reply:
Did you move this check to `TestMain()`?

Reply:
@jwendell 

Oh, sorry, I confused with the comment.
Thought it's the same as above.

I can't really move it to the `TestMain()`, because I need the `t framework.TestContext` available for the services and events check.